content/en/docs/tasks/configure-pod-container/configure-service-account.md

```diff
@@ -341,7 +341,7 @@ Now, when a new Pod is created in the current namespace and using the default
 ServiceAccount, the new Pod has its `spec.imagePullSecrets` field set automatically:
 
 ```shell
-kubectl run nginx --image=nginx --restart=Never
+kubectl run nginx --image=image-registry.some.example/nginx --restart=Never
```

Comment:
How about this? I mean, making it "fail" explicitly.

```suggestion
kubectl run nginx --image=<registry name>/nginx --restart=Never
```

Reply:
Whatever we pick, we should change this:
```shell
kubectl create secret docker-registry myregistrykey --docker-server=DUMMY_SERVER \
        --docker-username=DUMMY_USERNAME --docker-password=DUMMY_DOCKER_PASSWORD \
        --docker-email=DUMMY_DOCKER_EMAIL
```
to match.

Reply:
Using `<registry name>` is more clear.